src/main/java/com/google/sps/servlets/AvailabilityServlet.java

```diff
@@ -62,7 +52,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
         List<TimeRange> timeslots = new ArrayList<TimeRange>();
 
-        for (Tutor tutor : hardcoded) {
+        for (Tutor tutor : hardcoded.getSampleTutors()) {
```

Comment:
```suggestion
        for (Tutor tutor : SampleData.getSampleTutors()) {
```
Since all of the methods and data are static, I don't think you have to create a new instance of SampleData.

Reply:
Oh that makes it so much better, thanks!